src/components/profile-page/access-tokens/profile-access-tokens.tsx

```diff
@@ -120,7 +122,12 @@ export const ProfileAccessTokens: React.FC = () => {
                       />
                     </Col>
                     <Col xs='auto'>
-                      <IconButton icon='trash-o' variant='danger' onClick={() => selectForDeletion(token.created)} />
+                      <IconButton
+                        icon='trash-o'
+                        variant='danger'
+                        onClick={() => selectForDeletion(token.created)}
```

Comment:
useCallback

Reply:
The complete profile page uses this messy old code that ignores the fact that useCallback exists.
Guess why there's a branch `refactor-profile-page` basing on top of this PR. :D